sopel/bot.py

```diff
@@ -1030,12 +1034,11 @@ def error(
         self,
         trigger: Optional[Trigger] = None,
         exception: Optional[BaseException] = None,
-    ):
+    ) -> None:
         """Called internally when a plugin causes an error.
 
-        :param trigger: the ``Trigger``\\ing line (if available)
-        :type trigger: :class:`sopel.trigger.Trigger`
-        :param Exception exception: the exception raised by the error (if
+        :param trigger: the IRC line that caused the error (if available)
+        :param exception: the exception raised by the error (if
                                     available)
```

Comment:
Oops, I missed this one before (it needs rewrapping after removing the type in `:param:`)

```suggestion
        :param exception: the exception raised by the error (if available)
```

Not much of this PR touched multi-line `:param:` descriptions, so I must have glossed over it. 😅